tests/post/steps/conftest.py

```diff
@@ -68,3 +69,13 @@ def rerun_bootstrap(host, version):
 @then("the Kubernetes API is available")
 def verify_kubeapi_service(host):
     _verify_kubeapi_service(host)
+
+
+@given(parsers.parse("Pods with app label '{app}' are '{state}'"))
+def check_pod_state(host, app, state):
+    with host.sudo():
+        cmd = ("kubectl --kubeconfig=/etc/kubernetes/admin.conf"
+               " get pods -n kube-system --field-selector=status.phase={state}"
+               " --selector=k8s-app={app} --no-headers")
+        res = host.check_output(cmd.format(app=app, state=state))
+        assert len(res.strip()) > 0, "Not {} pod found".format(state)
```

Comment:
Minor [typo] IMO it is "No Running pod found" instead of "No**t** Running pod found"